echopype/preprocess/api.py

```diff
@@ -156,29 +170,32 @@ def compute_MVBS_index_binning(ds_Sv, range_sample_num=100, ping_num=100):
         .mean(skipna=True)
     )
 
-    # Attach coarsened echo_range
+    # Attach attributes and coarsened echo_range
     da.name = "Sv"
-    ds_out = da.to_dataset()
-    ds_out["echo_range"] = (
+    ds_MVBS = da.to_dataset()
+    ds_MVBS.coords["range_sample"] = (
+        "range_sample",
+        np.arange(ds_MVBS["range_sample"].size),
+        {"long_name": "Along-range sample number, base 0"},
+    )  # reset range_sample to start from 0
+    ds_MVBS["echo_range"] = (
         ds_Sv["echo_range"]
         .coarsen(  # binned echo_range (use first value in each average bin)
             ping_time=ping_num, range_sample=range_sample_num, boundary="pad"
         )
         .min(skipna=True)
     )
-    ds_out.coords["range_sample"] = (
-        "range_sample",
-        np.arange(ds_out["range_sample"].size),
-    )  # reset range_sample to start from 0
-
-    # Attach attributes
-    ds_out.attrs = {
-        "binning_mode": "index",
-        "range_sample_num": range_sample_num,
-        "ping_num": ping_num,
-    }
+    _set_MVVBS_attrs(ds_MVBS)
+    ds_MVBS["Sv"] = ds_MVBS["Sv"].assign_attrs(
+        {
+            "comment": "MVBS binned on the basis of range_sample and ping number specified as index numbers",
```

Comment:
@leewujung pay attention to this addition. I couldn't come up with a `cell_methods` alternative. I was getting hung up with the specification of bins based on index numbers rather than physical units.
FYI, `comment` is a CF attribute.